ROMFS/px4fmu_common/init.d/rc.gnd_defaults

```diff
@@ -0,0 +1,36 @@
+#!nsh
+
+set VEHICLE_TYPE ugv
+
+if [ $AUTOCNF == yes ]
+then
+#
+# Default parameters for FW
+#
+	param set RTL_RETURN_ALT 100
+	param set RTL_DESCEND_ALT 100
+	param set RTL_LAND_DELAY -1
+
+	# FW uses L1 distance for acceptance radius
+	#  set a smaller NAV_ACC_RAD for vertical acceptance distance
+	param set NAV_DLL_ACT 0
+	param set NAV_ACC_RAD 2.0
+	param set NAV_LOITER_RAD 5
+
+	param set MIS_LTRMIN_ALT 0.01
+	param set MIS_TAKEOFF_ALT 0.01
```

Comment:
That has been one of the major difficulties of working with a rover, i.e. trying to figure out how to just get rid of takeoff and landing without affecting other files. I started working on it 2-3 months ago and now navigator has changed so the new mission feasibility checker prevents me to add a takeoff waypoint with altitude 0. Now, even if I comment out that, the rover spins around the takeoff point 3-4 times and then moves to the second waypoint.